### PR TITLE
Group by science requirements

### PR DESCRIPTION
**Pull request by swalker2m**
Adds grouping by common science requirements.

### CODE REVIEW

File: modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala [hunk 1]
```diff
@@ -9,6 +9,7 @@ import lucuma.core.`enum`.{ObsActiveStatus, ObsStatus}
 import lucuma.core.optics.syntax.lens._
 import lucuma.core.model.{Asterism, Observation, Program, Target}
 import cats.{Eq, Monad}
+import cats.Order.catsKernelOrderingForOrder
```
Comment (swalker2m) [inline modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala:12]: I always need this but I'm never sure what the right way to import it is.

Comment (cquiroz) [reply modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala:12]: Maybe cats.implicits.order?

Comment (toddburnside) [reply modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala:12]: You can use `cats.implicits._` or `cats.implicits.catsKernelOrderingForOrder` I believe.

**Review by toddburnside — APPROVED** [review]
LGTM

**Pull request merged**